cmd/snap/cmd_sign.go

```diff
@@ -118,8 +119,11 @@ func (x *cmdSign) Execute(args []string) error {
 		return err
 	}
 
-	// --direct implies --chain
-	if x.Chain || x.Direct {
+	if x.Direct && !x.Chain {
+		return errors.New(i18n.G("signing with --direct requires specifying --chain"))
```

Comment:
this error is a bit confusing to me, as --direct doesn't really relate to signing itself. Maybe:

"--direct specified without --chain"

Reply:
Ack